doc/main.html

```diff
@@ -603,15 +605,20 @@ <h4>Technology Committee Chairs.</h4>
         <section id='Membership-and-Guests'>
           <h4>Membership and Guests</h4>
 
+          <section id="sec-basic-requirements">
+            <h5>Basic Requirements</h5>
           <p>
             All <a>Technology Committee</a> Members shall be Members
             (of the Standards Community). In addition, <a>Guests</a>
             may participate as defined further below.
           </p>
+          </section>
 
           <section id='Categories'>
-            <h4>Categories</h4>
+            <h5>Categories</h5>
 
+            <section id="sec-catagories-intro">
+              <h6>Introduction</h6>
```

Comment:
this should be "Overview"
```suggestion
              <h6>Overview</h6>
```